src/core/renderer.js

```diff
@@ -42,9 +43,20 @@ let corefn = ({
       return;
     }
 
-    let rOpts = util.extend( {}, options, {
-      cy: cy
-    } );
+    let defaults = {
+      motionBlur: false,
+      motionBlurOpacity: 0.05,
+      pixelRatio: undefined,
```

Comment:
pixelRatio had a undefined default value from before these changes.  Not sure if there is a more appropriate default value.

Reply:
You're right. The correct value is undefined. My mistake. It's because the 'auto'/nil value can only be calculated later by the renderer.

The change makes line 58 clearer, though it's not really necessary.